### DISCUSSION TITLE
Robotic voice as I train HiFiGAN vocoder

### DISCUSSION THREAD

**Discussion by KakoozaJerry** [coqui-ai/TTS#1008] [General Q&A]
Hello guys , so am trying to build a vocoder model with HiFiGAN , sofar I have trained up to 70k steps but I still get a robotic voice. Is there any hope that I will get better output .
Below are my output samples with the tacotron 2 model

https://soundcloud.com/jerry-kakooza/sets/tacotron2-ddc-and-custom

**Comment by skol101** [comment]
Could you share your training setup and screenshots? I'm trying to figure what's wrong with my setup when I'm trying to train HiFiGan https://github.com/coqui-ai/TTS/issues/1003

**Reply by KakoozaJerry** [reply]
Specifically for HiFiGAN, after installing coqui TTS, I initiate training using this command below

`CUDA_VISIBLE_DEVICES=0 python /path/to/train_hifigan.py`

And this is my train_hifigan.py file here [https://gist.github.com/KakoozaJerry/075792cafda5afd38e74a8240648e423](https://gist.github.com/KakoozaJerry/075792cafda5afd38e74a8240648e423)

**Reply by KakoozaJerry** [reply]
Set your `output_path`(path to where you want to set your output logs) and `data_path` (path to your wav files directory i,e ..../wavs)
I think you need to make sure the `mel_fmin`,`mel_fmax` and other audio_config parameters align with those of your acoustic model.

**Reply by skol101** [reply]
How are you able to train with target_loss = "avg_G_loss"? 
I get the KeyError: 'avg_G_loss', because default loss func for gan is "loss_0".

**Reply by KakoozaJerry** [reply]
Sorry . Try using this https://drive.google.com/file/d/18BNtFUah_HbdYMfdZKXXlsEb06JvdgeP/view?usp=drivesdk my updated .py file that worked

**Reply by TimofeyKruk** [reply]
@KakoozaJerry Hi! I have the same issue, that trained voice sounds robotic. Have you found the solution?
Thank you in advance!

**Reply by erogol** [reply]
Try the dev branch for anyone who experiences HiFiGAN issue
